components/personal-title.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { PersonalTitle } from './personal-title';
+
+export default {
+	title: 'PersonalTitle',
+	component: PersonalTitle,
+};
+
+export const Basic = (args) => <PersonalTitle {...args} />;
+Basic.args = {
+	options: [
+		{ code: 'Mr', description: 'Mr' },
+		{ code: 'Mrs', description: 'Mrs' },
+	],
+};
```

Comment:
Nitpick: if you keep Basic without args it will be rendered as default, and then you can add a second scenario with custom option values! You can validate the results on http://localhost:5005/?path=/docs/personaltitle--basic (`docs` tab)

```suggestion

export const CustomPersonalTitle = (args) => <PersonalTitle {...args} />;
CustomPersonalTitle.args = {
	options: [
		{ code: 'Mr', description: 'Mr' },
		{ code: 'Mrs', description: 'Mrs' },
	],
};
```

Reply:
Updated

Reply:
@PlamenHristov What I mean is to keep both:
- Basic without any customisation
- and another one with custom options

Something like this:
```Javascript
export const Basic = (args) => <PersonalTitle {...args} />;

export const CustomPersonalTitle = (args) => <PersonalTitle {...args} />;
CustomPersonalTitle.args = {
	options: [
		{ code: 'Mr', description: 'Mr' },
		{ code: 'Mrs', description: 'Mrs' },
	],
};
```